README.md

```diff
@@ -69,7 +69,10 @@ Installation
 
 We release `tttool` for Windows and Linux users as a zipfile. You can find them
 in the [releases section](https://github.com/entropia/tip-toi-reveng/releases)
-of the GitHub project.
+of the GitHub project.  
+For Linux users:
+- Install libttspico-utils: `sudo apt install libttspico-utils`
+- Install vorbis-tools: `sudo apt install vorbis-tools`
```

Comment:
```suggestion

Linux users likely want to install additional packages. On Debian or Ubuntu, run `sudo apt install libttspico-utils vorbis-tools`
```

Reply:
This might be easier to understand, what do you think? (Not all Linux users have `apt install`)